### PR TITLE
[Opt](Vec) Support const column check nullable and remove nullable

### PR DESCRIPTION
**Pull request by HappenLee**
# Proposed changes

1. Support const column check nullable
2. Support const column remove nullable

## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [ ] No
    - [ ] I don't know
3. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
4. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
5. Does it need to update dependencies:
    - [ ] Yes
    - [ ] No
6. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [ ] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: be/src/vec/columns/column_nullable.cpp [hunk 1]
```diff
@@ -570,6 +570,16 @@ ColumnPtr remove_nullable(const ColumnPtr& column) {
     if (is_column_nullable(*column)) {
         return reinterpret_cast<const ColumnNullable*>(column.get())->get_nested_column_ptr();
     }
+
+    if (is_column_const(*column)) {
```
Comment (xy720) [inline be/src/vec/columns/column_nullable.cpp:574]: Why not put this in a new method in column_const.h?

Comment (HappenLee) [reply be/src/vec/columns/column_nullable.cpp:574]: here two method `add_remove` nullable bosh in `nullable.cpp`

File: build.sh [hunk 2]
```diff
@@ -117,7 +117,7 @@ fi
 
 eval set -- "${OPTS}"
 
-PARALLEL="$(($(nproc) / 4 + 1))"
+PARALLEL="$(($(nproc) / 2 + 1))"
```
Comment (BiteTheDDDDt) [inline build.sh:120]: do not change it

**Review by BiteTheDDDDt — APPROVED** [review]
LGTM

**Pull request merged**